pkg/api/testapi/testapi.go

```diff
@@ -55,9 +55,10 @@ var (
 )
 
 type TestGroup struct {
-	externalGroupVersion unversioned.GroupVersion
-	internalGroupVersion unversioned.GroupVersion
-	internalTypes        map[string]reflect.Type
+	// the first element in the group is meant to be the preferred version
```

Comment:
Yes it does :/. My work is just a tip of an iceberg we need to dig into to get multiple versions fully testable.